src/aat/resources/features/F-064/S-218.td.json

```diff
@@ -0,0 +1,25 @@
+{
+  "title": "must return 403 when request provides authentic credentials without authorised access to the operation",
+	"_guid_": "S-218",
+	"_extends_": "F-064_Test_Data_Base",
+
+	"specs": [
+		"an active profile in CCD",
+		"provides authentic credentials without authorised access to the operation",
+		"code is HTTP-403 Forbidden"
+	],
+  "request": {
+    "headers": {
+      "ServiceAuthorization": "eyJhbGciOiJIUzI1NiJ9"
```

Comment:
This value does not actually meet the spec "provides authentic credentials without authorised access to the operation". Therefore, the scenario title/content and test data are inconsistent. However, we may not be able to actually play the scenario as per its original intent with the current API behaviour, either. I think we can adjust the scenario to the behaviour of the system now, keeping in mind that we already have some tickets opened to check and correct the system behaviour as per the original scenario design.

Reply:
updated with "Authorization" : "eyJhbGciOiJIUzI1NiJ9"